src/UI/Components/Diagram/interfaces.ts

```diff
@@ -159,6 +159,23 @@ interface StencilState {
   };
 }
 
+/**
+ * Interface representing a relation on the canvas with a minimum value.
+ */
+interface InterServiceRelationOnCanvasWithMin {
+  name: string;
+  min: ParsedNumber;
+  current: number;
```

Comment:
Ah, so this is where the current is coming from... 
For me, current is typically the current of an HTML Ref, which is an element.
Is there not a better term for this variable you need? To avoid confusion with HTML Semantics?

Reply:
currentAmount? I don't know, I wouldn't think of current in term of Ref, especially as it's not often used in our app.